qiita_pet/handlers/artifact_handlers/base_handlers.py

```diff
@@ -390,25 +391,33 @@ def artifact_post_req(user, artifact_id):
     """
     artifact_id = int(artifact_id)
     artifact = Artifact(artifact_id)
-    check_artifact_access(user, artifact)
```

Comment:
Any reason why the access doesn't need to be checked if the artifact is being deleted?

Reply:
Thank you for catching this!!! It was an extra delete!

Reply:
Just realized that it was under the new if and now is duplicated; will rm the duplicated line